lib/clearance/engine.rb

```diff
@@ -1,12 +1,19 @@
-require 'clearance'
-require 'rails'
+require "clearance"
+require "rails"
 
 module Clearance
   class Engine < Rails::Engine
-    initializer 'clearance.filter' do |app|
+    initializer "clearance.filter" do |app|
       app.config.filter_parameters += [:password, :token]
     end
 
-    config.app_middleware.insert_after ActionDispatch::ParamsParser, Clearance::RackSession
+    config.app_middleware.insert_after(
+      ActionDispatch::ParamsParser,
+      Clearance::RackSession
```

Comment:
@sanderhahn rubocop can re-write your code according to rules if you'd like: https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/corrector.rb